### ISSUE TITLE
Vector amo instruction support

### ISSUE THREAD

**Issue by hanfeng0114** [riscv/riscv-isa-sim#1260]
It seems the latest vector spec has excluded vector amo instructions.
Will spike excludes the instructions?

**Comment by aswaterman** [comment]
cc @chihminchao: we will probably need to remove these at some point anyway, since a future subword AMO extension is likely to use that opcode space.

**Comment by hanfeng0114** [comment]
Will the final solution be settled soon?

**Comment by aswaterman** [comment]
Spike will eventually exclude the instructions, yes. But one of the maintainers of the vector support needs to do the work.

**Comment by chihminchao** [comment]
I agree to remove them and we could dig out the implementation from log in future.

**Comment by hanfeng0114** [comment]
So will it be removed soon?

**Comment by chihminchao** [comment]
@hanfeng0114 Could you send a PR for that ?
